source/retryable-writes/tests/README.md

```diff
@@ -65,40 +65,8 @@ allow the fail point to trigger on the second command.
 
 ## Command Construction Tests
 
-Drivers should also assert that command documents are properly constructed with or without a transaction ID, depending
-on whether the write operation is supported.
-[Command Logging and Monitoring](../../command-logging-and-monitoring/command-logging-and-monitoring.md) may be used to
-check for the presence of a `txnNumber` field in the command document. Note that command documents may always include an
-`lsid` field per the [Driver Session](../../sessions/driver-sessions.md) specification.
-
-These tests may be run against both a replica set and shard cluster.
-
-Drivers should test that transaction IDs are never included in commands for unsupported write operations:
-
-- Write commands with unacknowledged write concerns (e.g. `{w: 0}`)
-- Unsupported single-statement write operations
-    - `updateMany()`
-    - `deleteMany()`
-- Unsupported multi-statement write operations
-    - `bulkWrite()` that includes `UpdateMany` or `DeleteMany`
-- Unsupported write commands
-    - `aggregate` with write stage (e.g. `$out`, `$merge`)
-
-Drivers should test that transactions IDs are always included in commands for supported write operations:
-
-- Supported single-statement write operations
-    - `insertOne()`
-    - `updateOne()`
-    - `replaceOne()`
-    - `deleteOne()`
-    - `findOneAndDelete()`
-    - `findOneAndReplace()`
-    - `findOneAndUpdate()`
-- Supported multi-statement write operations
-    - `insertMany()` with `ordered=true`
-    - `insertMany()` with `ordered=false`
-    - `bulkWrite()` with `ordered=true` (no `UpdateMany` or `DeleteMany`)
-    - `bulkWrite()` with `ordered=false` (no `UpdateMany` or `DeleteMany`)
+Command construction tests asserting on the absence or presence of the `txnNumber` field have been replaced by command
```

Comment:
I feel this wording is a little confusing - originally it made me think we are no longer asserting on `txnNumber`.

I think it'd be better to remove this section, and  if we want to include this information somewhere, we can expand in the changelog instead. Something like the following:

>  Convert command construction tests to unified format. In the retryable writes unified tests, the unified spec runner now asserts on the absence or presence of the `txnNumber` field using 'Command Monitoring'.

What do you think?

Reply:
The specs README states:

>Outdated tests must not be removed completely, but may be marked as such (e.g. by striking through or replacing the entire test with a note (e.g. Removed).

I think it's useful to maintain this section as a reference for drivers that have not yet removed these tests. WDYT about adding a strikethrough to the section title and rephrasing this as:

>These prose tests have been removed in favor of unified format tests.

Reply:
Sure, we can also rephrase it to something like.

> In the retryable writes unified tests, the unified spec runner now asserts on the absence or presence of the txnNumber field using 'Command Monitoring'.

This or the strikethrough works!